src/eval-frame/components/reps/rep-utils/get-top-level-rep-summary.js

```diff
@@ -0,0 +1,33 @@
+import { getObjAtPath } from "./get-child-summaries";
+import { serializeForValueSummary } from "./value-summary-serializer";
+import { getErrorStackString } from "./get-error-stack-summary";
+import { getInitialDataTableSummary } from "./get-data-table-summary";
+import { isRowDf } from "./rep-type-chooser";
+
+import UserReps from "../user-reps-manager";
+
+export function getTopLevelRepSummary(rootObjName, path) {
```

Comment:
this file/function produces the top-level summary needed to chose between different top level reps